docker-naming-convension.md

```diff
@@ -0,0 +1,26 @@
+#Docker Naming Conventions
```

Comment:
`# Docker Naming Conventions`

Reply:
I removed that file. See https://github.com/connexta/multi-int-store/pull/36#discussion_r292198808.